misc/SW_WA_2021_2024/guilderton_to_geraldton/swals/README.md

```diff
@@ -0,0 +1,390 @@
+Rscript make_initial_conditions_complex_historical_events.R
+# Then you can run the historical events
+qsub run/Sumatra2004_time_varying.pbs
+qsub run/Sumatra2004_time_varying_lowres.pbs
+qsub run/Sumatra2005.pbs
+qsub run/Sumatra2005_lowres.pbs
+
+# Run an extreme source, and a low-resolution version to check convergence in the inundation regime.
+qsub run/extreme_source.pbs
+qsub run/extreme_source_lowres.pbs
+
+# Run a very small source -- this can be useful to discover nesting artefacts.
+# (especially max_flux).
+qsub run/small_source.pbs
+
+#
+# After running the above jobs
+# 
+# You might want to make a folder to hold the *.sh.* files resulting from finished runs.
+# * I usually call this dot_e_o_files
+#
+# Check the load balancing again. 
+# * The efficiency of regular resolution models may be improved by making a new load_balance_file based on one of those runs. 
+# * The efficiency of low resolution runs might also be improved (with a separate load-balance file). 
+#
+# Check the log files for mass conservation, energy decay, reasonable peak stage/speed etc.
+# See ../analysis/check_log_files for more info.
+#
+# Make rasters in the usual way and check the results, e.g. on a node with 48 cores from inside the multidomain_dir
+Rscript ../../../post_process/make_rasters.R '.' 48 max_stage max_flux max_speed time_of_max_stage
+gdalbuildvrt -resolution highest max_stage_all.vrt max_stage_domain*.tif
+gdalbuildvrt -resolution highest max_speed_all.vrt  max_speed_domain_*.tif
+gdalbuildvrt -resolution highest time_of_max_stage_all.vrt  time_of_max_stage_domain_*.tif
+gdalbuildvrt -resolution highest max_flux_all.vrt  max_flux_domain_*.tif
+# If you saved temporal parameters you could additionally create the last timestep rasters
+gdalbuildvrt -resolution highest last_timestep_UH_all.vrt last_time_UH_domain_*.tif
+gdalbuildvrt -resolution highest last_timestep_VH_all.vrt last_time_VH_domain_*.tif
+#
+# Process gauges for the historical validation cases (two steps for each event, figures end up in the ./plots/ folder).
+# Example for Sumatra 2004
+cd plots
+Rscript process_gauges_sumatra2004.R ../OUTPUTS/path_to_multidomain_dir_sumatra2004
+cd ..
+# plot with varying y-scale
+Rscript plot_gauges_perth_sumatra2004.R OUTPUTS/path_to_multidomain_dir_sumatra2004/gauge_RDS_file_name.RDS
+# uniform y range of 1m
+Rscript plot_gauges_perth_sumatra2004.R OUTPUTS/path_to_multidomain_dir_sumatra2004/gauge_RDS_file_name.RDS 1.0
+
+# As above, Sumatra 2005
+cd plots
+Rscript plots/process_gauges_sumatra2005.R ../OUTPUTS/path_to_multidomain_dir_sumatra2005
+cd ..
+# varying y range
+Rscript plot_gauges_perth_sumatra2005.R OUTPUTS/path_to_multidomain_dir_sumatra2005/gauge_RDS_file_name.RDS
+# uniform y range of 0.3m
+Rscript plot_gauges_perth_sumatra2005.R OUTPUTS/path_to_multidomain_dir_sumatra2005/gauge_RDS_file_name.RDS 0.3
+#
+
+
+#
+# Run the random scenarios
+#
+
+# Set-up a script that can create all the PBS submission scripts, for all
```

Comment:
Somewhere here the user needs to cd into `pre_process` or else have the right paths to scripts.